### PR TITLE
Jtm draft status change (1.8.0)

### PR DESCRIPTION
**Pull request by BigChiefSmidgeums**

### CODE REVIEW

File: app/controllers/service_calendars_controller.rb [hunk 1]
```diff
@@ -127,6 +127,9 @@ def update
       else
         @line_items_visit.arm.line_items_visits
       end
+    @sub_service_request.update_attribute(:status, "draft") if @sub_service_request
+    @service_request.update_attribute(:status, "draft")
+    render partial: 'update_service_calendar'
```
Comment (amcates) [inline app/controllers/service_calendars_controller.rb]: please remove

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -187,9 +187,10 @@ def generate_visit_header_row arm, service_request, page, sub_service_request, p
...
                                       tag(:br) +
                                       link_to((content_tag(:span, '', class: "ui-button-icon-primary ui-icon #{icon}") + content_tag(:span, 'Check All', class: 'ui-button-text')),
-                                              "/service_requests/#{service_request.id}/#{action}/#{n}/#{arm.id}?portal=#{portal}",
+                                              "/service_requests/#{service_request.id}/toggle_calendar_column/#{n}/#{arm.id}?#{action}=true&portal=#{portal}",
                                               remote: true, role: 'button', class: 'ui-button ui-widget ui-state-default ui-corner-all ui-button-icon-only', id: "check_all_column_#{n}", data: ( visit_group.any_visit_quantities_customized?(service_request) ? { confirm: "This will reset custom values for this column, do you wish to continue?"} : nil)),
                                       width: 60, class: 'visit_number')
+      #binding.pry
```
Comment (amcates) [inline app/helpers/application_helper.rb]: please remove

**Comment by amcates** [comment]
Should 'complete' requests go back to draft in 'Admin'?
Does 'status' dropdown refresh in dashboard?

**Comment by kyle-glick** [comment]
Talked to Wenjun. She said this functionality should not be done in the dashboard

**Pull request merged**